src/cast_core/src/Loader.cpp

```diff
@@ -195,7 +195,7 @@ Loader::Loader(genny::ActorContext& context, uint thread, size_t totalThreads)
       _totalBulkLoad{context.operation("TotalBulkInsert", Loader::id())},
       _individualBulkLoad{context.operation("IndividualBulkInsert", Loader::id())},
       _indexBuild{context.operation("IndexBuild", Loader::id())},
-      _client{std::move(context.client())},
+      _client{std::move(context.client(context.get("ClientName").maybe<std::string>().value_or("Default")))},
```

Comment:
I didn't realize at first that the `Client` option was a `CrudActor` internal thing. Would it be worth adding this option to all actors?

Reply:
Looks like runCommand has it as well. I'm assuming people add this to their actor as they see the need. Up to you on whether to add or not, but would be a nice change

Reply:
Cool created TIG-3714 for this.